pkgs/applications/misc/hr/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "hr-${version}";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "LuRsT";
+    repo = "hr";
+    rev = version;
+    sha256 = "162vkip2772jl59lschpinimpg4ssiyg7fq0va5cx4d7wldpqmks";
+  };
+
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
```

Comment:
While it may work well here, I'm always suspicious about explicit phase listings. In this case, it'd break e.g., passing in additional patches via override. Disabling problematic phases if necessary is generally much more robust, from what I've seen.